apps/service-portal/src/components/Header/Header.tsx

```diff
@@ -96,21 +99,18 @@ export const Header = ({ position }: Props) => {
                       {user && <UserLanguageSwitcher user={user} />}
                       <Hidden below="md">
                         <Box marginRight={[1, 1, 2]} position="relative">
-                          <Link to={DocumentsPaths.ElectronicDocumentsRoot}>
+                          <LinkResolver
+                            href={DocumentsPaths.ElectronicDocumentsRoot}
+                          >
                             <Button
-                              variant="utility"
-                              colorScheme="white"
-                              size="small"
                               icon="mail"
                               iconType="outline"
-                              type="span"
-                              unfocusable
```

Comment:
You'll get a double focus here with the button as type button focusable within the link resolver.

Reply:
I changed the `LinkResolver `to a `Link`